libbeat/template/template.go

```diff
@@ -106,7 +106,13 @@ func (t *Template) generate(properties common.MapStr, dynamicTemplates []common.
 		"settings": common.MapStr{
 			"index.refresh_interval": "5s",
 		},
-		"template": t.GetName() + "-*",
+	}
+
+	// ES 6 moved from template to index_patterns: https://github.com/elastic/elasticsearch/pull/21009
```

Comment:
Nice to have the link to the github issue here. makes it easy to track the change later again.